code/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -1852,6 +1852,7 @@
 /datum/sprite_accessory/wings/skeleton
 	name = "Skeleton"
 	icon_state = "skele"
+	color_src = 0
```

Comment:
use boolean defines here, FALSE instead of 0

Reply:
done